src/pages/OpportunityDetails.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { MapPinIcon, CalendarDaysIcon, HeartIcon } from "@heroicons/react/24/outline";
+
+function OpportunityDetails() {
+  const opportunityData = {
+    title: "Green Guardians: Community Tree Planting Initiative",
+    location: "Los Angeles",
+    date: "August 30",
+    latitude: 34.0522,
+    longitude: -118.2437,
+    link: "https://google.com",
+    mainImageUrl: "../public/image1.webp",
```

Comment:
Nit: for local images in the public folder, you don't need to prefix with `/public`, so can just say `/image1.webp`

Reply:
Done!